super/CMakeLists.txt

```diff
@@ -42,7 +42,10 @@ include(ExternalProject)
 ExternalProject_Add(
     google_cloud_cpp_project
     DEPENDS ${GOOGLE_CLOUD_CPP_DEPENDENCIES_LIST}
-    EXCLUDE_FROM_ALL OFF
+    EXCLUDE_FROM_ALL
+        OFF
+        BUILD_ALWAYS
+        1
```

Comment:
Nit: Is that the desired formatting?

Reply:
No, but it is the formatting that cmake-format creates. I try to create a separate PR that teaches the robots to do better.